app/src/test/java/org/simple/clinic/summary/addphone/AddPhoneNumberDialogControllerTest.kt

```diff
@@ -79,13 +80,10 @@ class AddPhoneNumberDialogControllerTest {
   }
 
   @Test
-  @Parameters(method = "validation errors")
-  fun `when save is clicked, the number should not be saved if it's invalid`(
-      validationError: Result
-  ) {
+  fun `when save is clicked, the number should not be saved if it's invalid`() {
     // given
     val newNumber = "123"
-    whenever(validator.validate(newNumber, type = LANDLINE_OR_MOBILE)).thenReturn(validationError)
+    whenever(validator.validate(newNumber, type = LANDLINE_OR_MOBILE)).thenReturn(validationErrors.random())
```

Comment:
When we remove parameterized tests, we should be adding a separate test for each one of the removed parameter combinations instead of using a random value. Can we do that in this commit?

Reply:
I initially went with that approach and it seemed like a lot of code duplication. So using random felt like a good option, but will change back to having different tests.

Reply:
The problem with this is that it adds non-determinism to the tests, which makes the test suite unreliable. Code duplication is encouraged in tests to keep them isolated and focused.